controllers/articles.controller.js

```diff
@@ -63,9 +65,29 @@ const postArticleComment = (req, res, next) => {
   }
 };
 
+const patchArticleVotes = (req, res, next) => {
+  const articleId = req.params.article_id;
+  const newVotes = req.body.inc_votes;
+  selectArticleVotes(articleId)
```

Comment:
you can just do this in the SQL string.   (see next comment).  No need to query the db to get the existing number of votes.